packages/node/src/types.ts

```diff
@@ -455,3 +467,9 @@ export interface FsUtilInterface {
 export interface WithFs {
   fs?: FsUtilInterface
 }
+
+export interface WithPolicy {
+  policy?: Partial<LavaMoatPolicy<Resources>>
```

Comment:
```suggestion
  policy?: LavaMoatPolicy
```

The options for `makeGlobalsAttenuator` as in current `main` are kind of old and busted.

Also: `makeGlobalsAttenuator` is not a public API, so this stuff should live in `internal.ts`.  `internal.ts` can pull from `types.ts`, but not vice-versa.

Reply:
@boneskull I think I need some clarification on this, how `makeGlobalsAttenuator` being "old and busted" relate to the suggestion of turning `Partial<LavaMoatPolicy<Resources>>` into `LavaMoatPolicy`? And am I supposed to be surprised committing your suggestion as-is fails, or is it obvious and I am missing something?

Reply:
The point I was trying to make was that:

- We should not copy the types from the docstring (as in the `main` branch), because it is _wrong_. The `policy` option should be a `LavaMoatPolicy`; not a `Partial<LavaMoatPolicy>`.
- The docstring of `makeGlobalsAttenuator` does not follow the convention of defining a type a function's options 

...but you have already fixed this, so there's nothing more for you to do.

I just pulled your branch and didn't see any typechecking failure, fwiw.

Reply:
@sei358392